pkg/context/httprequest.go

```diff
@@ -96,8 +102,15 @@ func (r *httpRequest) Scheme() string {
 		return scheme
 	}
 
-	if scheme := r.std.Header.Get("X-Forwarded-Proto"); scheme != "" {
-		return scheme
+	if scheme := r.std.Header.Get(xForwardedProto); scheme != "" {
+		// note some proxy servers will append data in this format,
+		// and the last one is sent by the last proxy-server and should use for easegress's proxy
+		// header : {
+		//    some-key: "ele1, ele2, ele3"
+		// }
+		// https://github.com/spring-cloud/spring-cloud-gateway/ ProxyExchange.java#appendXForwarded()
+		schemes := strings.Split(scheme, headerValueSeparator)
```

Comment:
I'm fine with the change, but I think it is better to use `,` directly than define a `headerValueSeparator`, the code would be more readable in this case.